src/main/java/no/nav/pto/veilarbportefolje/postgres/BrukerRepositoryV2.java

```diff
@@ -252,10 +256,10 @@ private void flettInnDataFraPDL(ResultSet rs, OppfolgingsBruker bruker) {
                 .setUtenlandskAdresse(rs.getString("utenlandskAdresse"))
                 .setHarUkjentBosted(rs.getBoolean("harUkjentBosted"))
                 .setBostedSistOppdatert(toLocalDateOrNull(rs.getString("bostedSistOppdatert")))
-                .setSikkerhetstiltak(rs.getString("sikkerhetstiltak_type"))
-                .setSikkerhetstiltak_gyldig_fra(rs.getString("sikkerhetstiltak_gyldigfra"))
-                .setSikkerhetstiltak_gyldig_til(rs.getString("sikkerhetstiltak_gyldigtil"))
-                .setSikkerhetstiltak_beskrivelse(rs.getString("sikkerhetstiltak_beskrivelse"))
+                .setSikkerhetstiltak(showSikkerhetsTiltak ? rs.getString("sikkerhetstiltak_type") : "")
```

Comment:
Ok, I am confused :D setsikkerhetstiltak looks like it should be a certain type of sikkerhetstiltak (I dont know what types we have), but you send in a boolean here?

Reply:
this is probably not the best field name, it should be `sikkerhetstiltakType`